### PR TITLE
Implement google reCAPTCHA

### PR DESCRIPTION
**Pull request by anasdove2020**
- [x] Implement google reCAPTCHA

Reference:
https://polyrific.atlassian.net/browse/PTUR-15

### CODE REVIEW

File: js/subscribe.js [hunk 1]
```diff
@@ -9,12 +12,48 @@ document.addEventListener('DOMContentLoaded', function () {
...
-        document.getElementById('email-id').classList.remove('subscription-box-input-error');
-        document.getElementById('button-box-id').classList.remove('subscription-box-button-error');
-        document.getElementById('subscribe-error-id').innerHTML = "";
-        document.getElementById('subscribe-error-id').style.display = "none";
-        document.getElementById('subscribe-result-id').style.display = "block";
-        document.getElementById("email-id").value = "";
+        grecaptcha.ready(function () {
+          try {
+            grecaptcha.execute(siteKey, { action: 'submit' }).then(function () {
+              const data = {
```
Comment (ahmadilham) [inline js/subscribe.js:18]: We need to include the generated token when submitting the form.

**Review by ahmadilham — CHANGES_REQUESTED** [review]

File: js/subscribe.js (hunk 1 above)
Comment (anasdove2020) [reply js/subscribe.js]: Tweak the code.

File: _config.production.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+env:
+  SITE_KEY: 
+  API_URL:
```
Comment (ahmadilham) [inline _config.production.yml:3]: It is better to skip this file for now

File: _config.dev.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+env:
+  SITE_KEY: 6Ldlk1MlAAAAAOIHWIHiQvisOoAs-CGSjAZFSoJ5
+  API_URL:
```
Comment (ahmadilham) [inline _config.dev.yml:3]: It is better to skip this file for now

**Review by ahmadilham — CHANGES_REQUESTED** [review]

**Review by ahmadilham — APPROVED** [review]
LGTM

**Pull request merged**